### PR TITLE
adding polygon examples

### PR DESCRIPTION
**Pull request by adrianoamaral**

### CODE REVIEW

File: docs/commands/ft.search.md [hunk 1]
```diff
@@ -478,6 +478,51 @@ To sum up, the `INORDER` argument or `$inorder` query attribute require the quer
 
 </details>
 
+<details open>
+<summary><b>NEW!!! Polygon Search with WITHIN and CONTAINS operators</b></summary>
+
+Search if a polygon contain or within inside a given geometry
```
Comment (oshadmi) [inline docs/commands/ft.search.md:484]:
```suggestion
Query for polygons which contain a given geometry or are within a given geometry

```

File: docs/commands/ft.search.md [hunk 2]
```diff
@@ -478,6 +478,51 @@ To sum up, the `INORDER` argument or `$inorder` query attribute require the quer
...
+
+{{< highlight bash >}}
+127.0.0.1:6379> FT.SEARCH idx '@geom:[WITHIN POLYGON((0 0, 0 150, 150 150, 150 0, 0 0))]' DIALECT 3
+1) (integer) 1
+2) "small"
+3) 1) "geom"
+   2) "POLYGON((1 1, 1 100, 100 100, 100 1, 1 1))"
+{{< / highlight >}}
+
+Query with `CONTAIN` operator:
```
Comment (oshadmi) [inline docs/commands/ft.search.md:511]:
```suggestion
Query with `CONTAINS` operator:

```

File: build/docker/Makefile [hunk 3]
```diff
@@ -7,7 +7,7 @@ REPO=redisearch
 REDIS_VERSION=7.2-rc1
 OSNICK.official=bullseye
 
-INT_BRANCHES=2.6 2.4 2.2 2.0 1.6
+INT_BRANCHES=2.8 2.6 2.4 2.2 2.0 1.6
```
Comment (oshadmi) [inline build/docker/Makefile:10]: Why is this change part of this documentation PR?

File: src/version.h [hunk 4]
```diff
@@ -10,9 +10,9 @@
 
 #pragma once
 
-#define REDISEARCH_VERSION_MAJOR 99
-#define REDISEARCH_VERSION_MINOR 99
-#define REDISEARCH_VERSION_PATCH 99
+#define REDISEARCH_VERSION_MAJOR 2
+#define REDISEARCH_VERSION_MINOR 8
+#define REDISEARCH_VERSION_PATCH 0
```
Comment (oshadmi) [inline src/version.h:15]: Should not be part of this PR
(master should remain with version 99.99.99)

**Review by oshadmi — COMMENTED** [review]
👍🏼  Some comments

File: build/docker/Makefile (hunk 3 above)
Comment (adrianoamaral) [reply build/docker/Makefile:10]: Maybe something automatic Omer, I didn't touch this part

File: src/version.h (hunk 4 above)
Comment (adrianoamaral) [reply src/version.h:15]: I didn't touch this section either, strange

File: docs/commands/ft.search.md [hunk 5]
```diff
@@ -478,6 +478,53 @@ To sum up, the `INORDER` argument or `$inorder` query attribute require the quer
 
 </details>
 
+<details open>
+<summary><b>NEW!!! Polygon Search with WITHIN and CONTAINS operators</b></summary>
+
+Query for polygons which contain a given geometry or are within a given geometry
+
+First, create an index using `GEOMETRY`type:
```
Comment (oshadmi) [inline docs/commands/ft.search.md:486]:
```suggestion
First, create an index using `GEOMETRY` type:

```

File: docs/commands/ft.search.md [hunk 6]
```diff
@@ -478,6 +478,53 @@ To sum up, the `INORDER` argument or `$inorder` query attribute require the quer
...
+Query for polygons which contain a given geometry or are within a given geometry
+
+First, create an index using `GEOMETRY`type:
+
+{{< highlight bash >}}
+127.0.0.1:6379> FT.CREATE idx SCHEMA geom GEOMETRY
+OK
+{{< / highlight >}}
+
+Adding couple geometries using `HSET`:
```
Comment (oshadmi) [inline docs/commands/ft.search.md:493]:
```suggestion
Adding a couple of geometries using `HSET`:

```

**Review by oshadmi — APPROVED** [review]
